### ISSUE TITLE
P1: distinguish evaluation-only evidence from training-eligible and hard-negative data

### ISSUE THREAD

**Issue by James3014** [James3014/nexus-learning#21]
## Status / priority

**P1 — DATA-PURPOSE / TRAINING-ADMISSION SAFETY GAP. `AUTO_CHAIN=false`.**

Parent research tracker: James3014/Nexus-new#1053.

Fresh source fence at Issue creation:

- repository: `James3014/nexus-learning`
- default branch: `main`
- observed HEAD: `c0c78203175441cf3821ab548201d1b4d6e040d9`
- relevant source:
  - `nexus_learning/contracts.py::LearningExperience`
  - `nexus_learning/contracts.py::project_model_training`
  - `nexus_learning/contracts.py::apply_autodata_quality_gate`

## Problem

Nexus Learning already separates learning/adaptation authority from route, worker, merge, release, and production authority.

It also has an explicit model-training projection:

```python
model_training_targets = ("preference_pair", "reward_row")
```

and current `project_model_training()` maps an ineligible experience to:

```text
training_eligible = false
targets = ["hard_negative"]
```

That is a useful fail-closed training-quality behavior, but it conflates two different states:

1. **not training-grade yet** — the material may still be eligible for a negative/training projection after quality gating; and
2. **training-forbidden by provenance/purpose** — the material must never become a label, pseudo-label, reward, preference pair, hard negative, distillation input, imitation target, or fine-tuning input.

The current contract does not expose a canonical purpose/provenance field that can express the second state before downstream training projection.

## Donor evidence

Semantic Reflex V2 Wave 1 used third-party hosted model outputs as private evaluation evidence only.

The experiment contract explicitly prohibited using provider outputs as:

- labels;
- pseudo-labels;
- rewards;
- preference data;
- distillation data;
- imitation data;
- fine-tuning data.

Those outputs were valid research evidence but intentionally **not training data**, including not `hard_negative`.

This is donor evidence for a generic data-purpose boundary. It is not authority to add provider-specific semantics to Learning.

## Goal

Add the smallest canonical data-purpose / training-admission boundary so Nexus Learning can distinguish:

```text
EVALUATION_ONLY / TRAINING_FORBIDDEN
from
TRAINING_CANDIDATE_BUT_NOT_YET_ELIGIBLE
from
TRAINING_ELIGIBLE
```

Prefer one canonical provenance/admission contract consumed by model-training projection rather than producer-specific `do_not_train` flags.

### G0 — reuse-before-extension proof

First prove whether an existing current field or validator can already express an immutable training prohibition such that:

- `project_model_training()` cannot emit `hard_negative`;
- `apply_autodata_quality_gate()` cannot later re-enable training;
- downstream projections can inspect the reason without reading producer-specific payloads.

If that is already possible, close with tests/docs and no new schema.

### G1 — minimal extension if G0 fails

If current contracts cannot express it, add a backward-compatible purpose/admission field or equivalent canonical contract.

Possible semantics are illustrative only; implementation may choose better names:

```text
data_purpose:
  - LEARNING_POLICY_EVIDENCE
  - EVALUATION_ONLY
  - TRAINING_CANDIDATE

training_policy:
  - FORBIDDEN
  - QUALITY_GATED
  - ELIGIBLE
```

Do not create overlapping authorities if one dimension is sufficient.

## Required invariants

- explicit `TRAINING_FORBIDDEN` always dominates generic training projection;
- forbidden evidence never becomes `hard_negative`;
- quality gates may narrow eligibility but cannot widen an immutable training prohibition;
- producer/model output cannot self-authorize training use;
- provenance/purpose survives normalization and projection;
- legacy experiences without the new field preserve backward compatibility under a documented default;
- Learning remains advisory and does not initiate model training itself.

## Acceptance tests

At minimum prove:

1. evaluation-only evidence produces no training target;
2. evaluation-only failed outcomes do not become `hard_negative`;
3. training-forbidden survives `apply_autodata_quality_gate()`;
4. a positive quality row cannot override training prohibition;
5. ordinary legacy training candidates retain current quality-gated behavior;
6. malformed/unknown purpose fails closed according to the chosen compatibility contract;
7. projections expose a machine-readable exclusion reason;
8. current learning-policy recommendation/adoption behavior remains unchanged.

## Non-goals

- no training pipeline implementation;
- no fine-tuning job execution;
- no provider-specific Jev rule;
- no route/model promotion;
- no weakening of existing AutoData leakage/reward-hacking gates.

## Claim ceiling

This Issue may establish **training-data admissibility semantics** for Nexus Learning. It does not determine whether any model should be trained, promoted, routed, or deployed.
